### DISCUSSION TITLE
Unused spawning waypoint?

### DISCUSSION THREAD

**Discussion by Quintus24** [AmProsius/gothic-1-community-patch#353] [Q&A] [answered]
Hello, everyone!

I stumbled upon this waypoint while fooling around in the game, **SPAWN_OW_BLOODHOUND_01**.
I was curious if anyone knows if it's some kind of leftover spawning waypoint, since I never noticed **Bloodhounds** in that area.
It's located on the route to the first **Focus Stone**, the one above the **Swamp Camp**.

![SPAWN_OW_BLOODHOUND_01](https://user-images.githubusercontent.com/93062373/170870675-348dcb62-dc43-43c9-a252-5181bde366d7.JPG)

Thank You and Best Regards!
Quintus24

**Comment by szapp** [comment, marked as answer]
Hi! Interesting find! Digging into the resources of the alpha state of the game may reveal the original intention, whether a potential bloodhound was forgotten, or whether it was purposely removed during development. If I had to take a guess, I would assume the latter; a bloodhound may have been intended there, but was removed due to limited strength of the player character at that point in the story. As we cannot (do not) know the intentions around the circumstances and because it does not "fix" any substantial bug or issue, we will most likely not take any action within the scope of the G1CP.

**Reply by Quintus24** [reply]
Noted, thank you! :star:
